Packs/FeedMandiant/Integrations/FeedMandiant/FeedMandiant.py

```diff
@@ -0,0 +1,655 @@
+              'targets': [target.get('name') for target in  # type:ignore
+                          raw_indicator.get('locations', {}).get('target', [])],  # type:ignore
+              'stixid': raw_indicator.get('id'),
+              'name': raw_indicator.get('name'),
+              'description': raw_indicator.get('description'),
+              'updateddate': raw_indicator.get('last_updated'),
+              'trafficlightprotocol': client.tlp_color
+              }
+
+    fields = {k: v for k, v in fields.items() if v and v != 'redacted'}  # filter none and redacted values
+
+    relationships = get_indicator_relationships(raw_indicator, 'malware', 'name', ThreatIntel.ObjectsNames.THREAT_ACTOR,
+                                                'name',
+                                                ThreatIntel.ObjectsNames.MALWARE,
+                                                EntityRelationship.Relationships.RELATED_TO,
+                                                EntityRelationship.Relationships.RELATED_TO)
+
+    relationships += get_indicator_relationships(raw_indicator, 'cve', 'name', ThreatIntel.ObjectsNames.THREAT_ACTOR,
+                                                 'cve_id',
+                                                 FeedIndicatorType.CVE,
+                                                 EntityRelationship.Relationships.TARGETS,
+                                                 EntityRelationship.Relationships.TARGETED_BY)
+
+    indicator_obj = {
+        'value': raw_indicator.get('name'),
+        'type': ThreatIntel.ObjectsNames.THREAT_ACTOR,
+        'rawJSON': raw_indicator,
+        'score': get_verdict(raw_indicator.get('mscore')),
+        'fields': fields,
+        'relationships': relationships
+    }
+    return indicator_obj
+
+
+def create_indicator(client: MandiantClient, raw_indicator: Dict) -> Dict:
+    """
+    Create indicator
+    Args:
+        client (MandiantClient): client
+        raw_indicator (Dict): raw indicator
+
+    Returns: Parsed indicator
+    """
+    fields = {'primarymotivation': raw_indicator.get('motivations'),
+              'firstseenbysource': raw_indicator.get('first_seen'),
+              'lastseenbysource': raw_indicator.get('last_seen'),
+              'stixid': raw_indicator.get('id'),
+              'trafficlightprotocol': client.tlp_color
+              }
+
+    fields = {k: v for k, v in fields.items() if v and v != 'redacted'}  # filter none and redacted values
+    indicator_obj = {
+        'value': raw_indicator.get('value'),
+        'type': MAP_INDICATORS_TYPE[raw_indicator.get('type', '')],
+        'rawJSON': raw_indicator,
+        'score': get_verdict(raw_indicator.get('mscore')),
+        'fields': fields
+    }
+    return indicator_obj
+
+
+MAP_INDICATORS_FUNCTIONS = {
+    'Malware': create_malware_indicator,
+    'Actors': create_actor_indicator,
+    'Indicators': create_indicator
+}
```

Comment:
move to the top of the file

Reply:
Impossible, ```MAP_INDICATORS_FUNCTIONS``` can only appear after the declarations of the functions.

Reply:
True 👍